tinygrad/runtime/lib.py

```diff
@@ -50,6 +54,13 @@ def _buffer(self) -> memoryview: raise NotImplementedError("must be implemented"
   # NOTE: this metadata prevents the backing buffer from being freed. hack can be removed with PEP688
   def toCPU(self) -> np.ndarray: return np.frombuffer(self._buffer(), dtype=np.dtype(self.dtype.np, metadata={"backing": self}), count=self.size)  # type: ignore
   def _copyin(self, x:np.ndarray) -> None: np.copyto(self.toCPU(), x.reshape(-1))
+  @classmethod
+  def from_buffer(cls, buffer: RawBuffer, _, **kwargs):
+    if isinstance(buffer.realized, RawDiskBuffer):
```

Comment:
Could you try moving `from tinygrad.runtime.ops_disk import RawDiskBuffer` right here to fix circular imports?

Reply:
It did the trick, thank you!